README.md

```diff
@@ -158,6 +158,12 @@ If you are looking for the codebase of the current production version of GATK, p
   on a cluster or the cloud by building a Spark jar with `gradle installSpark` and passing the resulting jar in `build/libs/`
   directly to either `spark-submit` or `gcloud`.
 
+* To pass JVM arguments to GATK (non-spark), use `JAVA_OPTS` like in this example:
```

Comment:
given that we recommend using gatk-launch i think spark-submit is not relevant in our readme